modules/installation-aws-tested-machine-types.adoc

```diff
@@ -41,14 +41,15 @@ ifdef::localzone[]
 .Machine types based on x86_64 architecture for AWS Local Zones
 [%collapsible]
 ====
-* `c5.2xlarge`
-* `c5d.2xlarge`
-* `m5.xlarge`
-* `m5.2xlarge`
-* `t3.xlarge`
+* c5.*
```

Comment:
Not sure if there was a reason to remove the backticks, but I think that most of the other lists like this have the types in backticks. (Including the table in the same assembly)

Reply:
Good catch. I updated each table entry to include backticks.